compiler/rustc_passes/src/errors.rs

```diff
@@ -563,7 +563,10 @@ pub struct ReprConflictingLint;
 #[diag(passes_used_static)]
 pub struct UsedStatic {
     #[primary_span]
+    pub attr_span: Span,
+    #[label]
     pub span: Span,
+    pub target: String,
```

Comment:
Suggestion: I believe this can be `&'static str` because we can use [`Target::name`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/enum.Target.html#method.name).

Reply:
> Suggestion: I believe this can be `&'static str` because we can use [`Target::name`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/enum.Target.html#method.name).

Done. Thank you.